generatebundlefile/data/bundles_staging/1-25-regional.yaml

```diff
@@ -72,7 +72,7 @@ packages:
         repository: cluster-autoscaler/charts/cluster-autoscaler
         registry: 067575901363.dkr.ecr.us-west-2.amazonaws.com
         versions:
-            - name: 9.21.0-1.25-93dbb0e58f98edc2cf246127109c680c0d7fec5c
+            - name: 9.34.0-1.25-93dbb0e58f98edc2cf246127109c680c0d7fec5c
```

Comment:
can we confirm the tag with `93dbb0e58f98edc2cf246127109c680c0d7fec5c` exists in `cluster-autoscaler/charts/cluster-autoscaler` in the 857 account? It won't be in 067 right now due to what we previously discussed.

I think we want to see whatever the latest dev build tag for 1-25 is and use that.